### ISSUE TITLE
Warming the RDF indexes results in a stack trace

### ISSUE THREAD

**Issue by wouterbeek** [SWI-Prolog/packages-semweb#44]
Warming the RDF indexes results in a stack trace:

```prolog
?- [library(semweb/rdf11)].
?- rdf_warm_indexes.
[Thread 1 (main) at Tue Aug  9 00:49:01 2016] rdf_db.c:3889: initial_size_triple_hash: Assertion failed: 0
C-stack trace labeled "assert_fail":
  [0] save_backtrace() at /home/wbeek/Git/swipl-devel/src/os/pl-cstack.c:322 [0x7fbfc904938a]
  [1] __assert_fail() at /home/wbeek/Git/swipl-devel/src/pl-thread.c:6513 [0x7fbfc900651a]
  [2] initial_size_triple_hash() at /home/wbeek/Git/swipl-devel/packages/semweb/rdf_db.c:3889 [0x7fbfc056155f]
  [3] rdf_warm_indexes() at /home/wbeek/Git/swipl-devel/packages/semweb/rdf_db.c:9326 [0x7fbfc05619e7]
  [4] PL_next_solution() at /home/wbeek/Git/swipl-devel/src/pl-vmi.c:3627 [0x7fbfc8fa5949]
  [5] query_loop() at /home/wbeek/Git/swipl-devel/src/pl-pro.c:108 [0x7fbfc8fdae7f]
  [6] prologToplevel() at /home/wbeek/Git/swipl-devel/src/pl-pro.c:400 [0x7fbfc8fdb63b]
  [7] PL_toplevel() at /home/wbeek/Git/swipl-devel/src/pl-fli.c:4154 [0x7fbfc8f9742d]
  [8] swipl(main+0x2d) [0x40084d]
  [9] __libc_start_main() at ??:? [0x7fbfc89a9731]
  [10] swipl(_start+0x29) [0x400899]
```

**Comment by JanWielemaker** [comment]
How did you call it?  Does it need specific data loaded?

**Comment by wouterbeek** [comment]
No, starting SWI, loading the library, and then calling `rdf_warm_indexes/0` consistently results in a stack trace.  Does it not reproduce on your machine?

**Comment by JanWielemaker** [comment]
I see.  Fixed in 8f6c8955360ccbebe79147a126e537777751ffc3
